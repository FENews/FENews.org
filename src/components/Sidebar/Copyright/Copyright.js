import React from 'react';
import styles from './Copyright.module.scss';

const Copyright = ({ copyright }) => (
  <div className={styles['copyright']}>
    {copyright}
  </div>
);

export default Copyright;