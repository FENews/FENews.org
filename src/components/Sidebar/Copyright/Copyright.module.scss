@import '../../../assets/scss/variables';
@import '../../../assets/scss/mixins';

.copyright {
  color: lighten($color-gray, 18%);
  font-size: $typographic-small-font-size;
}