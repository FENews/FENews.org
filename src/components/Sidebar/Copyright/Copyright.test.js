import React from 'react';
import renderer from 'react-test-renderer';
import Copyright from './Copyright';

describe('Copyright', () => {
  it('renders correctly', () => {
    const props = {
      copyright: 'copyright'
    };

    const tree = renderer.create(<Copyright {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
