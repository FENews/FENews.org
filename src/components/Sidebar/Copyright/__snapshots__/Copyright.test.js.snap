// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Copyright renders correctly 1`] = `
<div
  className="copyright"
>
  copyright
</div>
`;
